packages/server/lib/util/settings.js

```diff
@@ -126,7 +126,11 @@ module.exports = {
         throw err
       }
 
-      // else we cannot read due to folder permissions
+      if (err.toString().startsWith('Error: EACCES: permission denied')) {
```

Comment:
See how line 119 uses an object to filter the `.catch` block. You should be able to do this in a similar fashion:

```js
.catch({ code: 'EACCES' }, () => {
	return errors.warning(...)
})
```

Reply:
done! thanks for the advise